Frontend-Web/contexts/UserContext.tsx

```
@@ -0,0 +1,127 @@
+
+    const login = async (email: string, password: string) => {
+        try {
+            const response = await apiService.login({ email, password });
+
+            setUser(response.user);
+            setToken(response.token);
+
+            localStorage.setItem("token", response.token);
+            localStorage.setItem("user", JSON.stringify(response.user));
+        } catch (error) {
+            throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)